wrapper/src/main/java/software/amazon/jdbc/dialect/DialectCodes.java

```diff
@@ -20,9 +20,11 @@ public class DialectCodes {
   public static final String AURORA_MYSQL = "aurora-mysql";
   public static final String RDS_MYSQL = "rds-mysql";
   public static final String MYSQL = "mysql";
+  public static final String RDS_MULTI_AZ_MYSQL_CLUSTER = "rds-multi-az-mysql-cluster";
```

Comment:
NIT: Should we add a comment on the difference between this new line and RDS_MYSQL?